namespaces/live.cloud-platform.service.justice.gov.uk/wiggedout-prod/resources/serviceaccount.tf

```diff
@@ -0,0 +1,80 @@
+module "serviceaccount" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-serviceaccount?ref=1.1.0"
+
+  namespace = var.namespace
+  kubernetes_cluster = var.kubernetes_cluster
+
+  serviceaccount_token_rotated_date = "01-01-2000"
```

Comment:
Could you have a look at the service account rotation date set? could pose a security risk